client/coral-admin/src/graphql/utils.js

```diff
@@ -66,21 +66,27 @@ function addCommentToQueue(root, queue, comment, sort) {
 
 function getCommentQueues(comment) {
   const queues = ['all'];
+
   if (comment.status === 'ACCEPTED') {
     queues.push('accepted');
   }
   else if (comment.status === 'REJECTED') {
     queues.push('rejected');
   }
-  else if (comment.status === 'PREMOD') {
-    queues.push('premod');
-  }
-  if (
+  else if (
```

Comment:
This should remain an `if` condition instead of `else if` (see explanation below)